src/Raven.Server/Config/Categories/SubscriptionConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+using Raven.Server.Config.Attributes;
+using Raven.Server.Config.Settings;
+
+namespace Raven.Server.Config.Categories
+{
+    public class SubscriptionConfiguration : ConfigurationCategory
+    {
+        [DefaultValue(10)]
```

Comment:
I didn't try to "stress" it yet. But we did want to limit the number of concurrent connections, maybe 10 is a bit low for a default value.